docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/10-advanced-with-helm.md

```diff
@@ -5,23 +5,26 @@ description: Learn how to perform an advanced installation of Kargo using Helm
 
 # Advanced Installation with Helm
 
-For a customized and secure Kargo setup, follow these advanced installation steps.
+For a customized Kargo setup, follow these generalized instructions for installing Kargo with non-default configuration.
```

Comment:
I try to stay away from "setup" because it feels vague.

```suggestion
This section outlines the general steps for customizing Kargo's configuration
when installing it via Helm:
```